GoogleUtilities/Network/Public/GoogleUtilities/GULNetworkURLSession.h

```diff
@@ -29,7 +29,7 @@ typedef void (^GULNetworkURLSessionCompletionHandler)(NSHTTPURLResponse *_Nullab
                                                       NSError *_Nullable error);
 typedef void (^GULNetworkSystemCompletionHandler)(void);
 
-/// The protocol that uses NSURLSession for iOS >= 7.0 to handle requests and responses.
+/// The type that wraps `NSURLSession` to handle requests and responses.
```

Comment:
Revert because there is some context there that this was added for some ios > 7 reason
```suggestion
/// The protocol that uses NSURLSession for iOS >= 7.0 to handle requests and responses.
```